API-Samples/CMakeLists.txt

```diff
@@ -223,16 +223,17 @@ if (NOT WIN32)
     if (NOT (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_CURRENT_BINARY_DIR))
         add_custom_target(samples-binary-dir-symlinks ALL
             COMMAND ln -sf ${CMAKE_CURRENT_SOURCE_DIR}/run_all_samples.sh
+			COMMAND ln -sf ${CMAKE_CURRENT_SOURCE_DIR}/run_all_samples.py
```

Comment:
indentation looks funny here

Reply:
This will work (copying the existing `ln` command) since you're inside a platform check.

The CMake-official way is `COMMAND ${CMAKE_COMMAND} -E create_symlink ${target} ${link}` see [here](https://cmake.org/cmake/help/v3.10/manual/cmake.1.html#command-line-tool-mode) which still needs a platform check

@lunarpapillo FWIW there is a [cmake_format](https://github.com/cheshirekow/cmake_format/) I've used time-to-time to get things all consistent, but it's not needed often enough to add to our standard process.